pom.xml

```diff
@@ -706,6 +707,7 @@
         <version>0.13.1</version>
         <configuration>
           <parameter>
+            <ignoreMissingOldVersion>true</ignoreMissingOldVersion>
```

Comment:
this was added as otherwise it would fail as there can be no previous version found of `resolver-dns-native-macos`.